data/assets/scene/digitaluniverse/starorbits.asset

```diff
@@ -197,12 +197,9 @@ asset.export(PM_J13420Orbit)
 
 asset.meta = {
   Name = "Star Orbits",
-  Version = "1.1",
-  Description = [[Select Star Orbital paths that delineate their trajectory around the
-    Milky Way over 1 billion years into the future. Included: Sun, Barnards, Kapteyns,
-    Lacaille 9352, LSR1826+3014, LSRJ0822+1700, PM_J13420-3415. <br><br> Data
-    Reference: Sebastien Lepine (AMNH)]],
-  Author = "Brian Abbott (AMNH)",
+  Version = "1.06",
+  Description = [[Projected star orbits for select stars over the next 1 billion years.]],
```

Comment:
Why removing the longer description?

Reply:
The idea with these descriptions, which I wrote to be more consistent and under the assumption that the user would access them in the GUI, are designed to convey a few things: a general description of the data set, a scientific description of the objects where applicable, and in some cases what the user will see (colors and such). References and such don't seem appropriate to me in that particular Asset info box. They are appropriate elsewhere, and we need to discuss that for a future version. I'd like to know where exactly all these descriptions are revealed, and how the info is used. The one bit of info I'd agree on is the census info, but I was hoping there would be some formatting available in those asset info boxes (new lines, etc.), but there doesn't appear to be. The description in many cases did not look clean to me. So, these changes are an effort to make the descriptions that appear in the asset info box more consistent informationally and stylistically, and to hopefully build upon that if there are formatting options.